README.md

```diff
@@ -85,27 +81,30 @@ representing the user choice (resolved on confirmation and rejected on cancellat
+| **`cancellationButtonProps`**           | `object`    | `{}`                                                                                         | Material-UI [Button](https://mui.com/material-ui/api/dialog/#props) props for the cancellation button.                                                                                                                                 |
+| **`titleProps`**                        | `object`    | `{}`                                                                                         | Material-UI [DialogTitle](https://mui.com/api/dialog-title/#props) props for the dialog title.                                                                                                                                         |
+| **`contentProps`**                      | `object`    | `{}`                                                                                         | Material-UI [DialogContent](https://mui.com/api/dialog-content/#props) props for the dialog content.                                                                                                                                   |
+| **`allowClose`**                        | `boolean`   | `true`                                                                                       | Whether natural close (escape or backdrop click) should close the dialog. When set to `false` force the user to either cancel or confirm explicitly.                                                                                   |
+| **`confirmationKeyword`**               | `string`    | `undefined`                                                                                  | If provided the confirmation button will be disabled by default and an additional textfield will be rendered. The confirmation button will only be enabled when the contents of the textfield match the value of `confirmationKeyword` |
+| **`confirmationKeywordTextFieldProps`** | `object`    | `{}`                                                                                         | Material-UI [TextField](https://mui.com/material-ui/api/text-field/) props for the confirmation keyword textfield.                                                                                                                     |
+| **`getConfirmationKeywordPlaceholder`** | `function`  | `getConfirmationKeywordPlaceholder: (keyword) => 'Please type "' + keyword + '" to confirm'` | Custom formatting for the placeholder in the confirmation keyword textfield.                                                                                                                                                           |
```

Comment:
We shouldn't show the keyword in the placeholder, because it can be long and as soon as the user types the first letter we no longer show the keyword. Since the caller can specify custom dialog description/content, I think we can leave it up to them to show the keyword. wdyt?

Reply:
Sounds good.